Extractor/UAClient.cs

```diff
@@ -104,6 +106,10 @@ public UAClient(IServiceProvider provider, FullConfig config)
             {
                 metricsManager = new NodeMetricsManager(this, config.Source, config.Metrics.Nodes);
             }
+            serverSubscription = new ServerSubscriptionManager(
```

Comment:
We probably do not need to create this if the system for subscribing to changes is not enabled.